backend/fms_core/migrations/0022_v3_4_0.py

```diff
@@ -1,48 +1,39 @@
-# Generated by Django 3.1 on 2021-03-02 18:25
+# Generated by Django 3.1 on 2021-08-04 17:59
+
 from django.conf import settings
+import django.core.validators
 from django.db import migrations, models
 import django.db.models.deletion
+import re
+
 
 class Migration(migrations.Migration):
 
     dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('fms_core', '0021_v3_3_0'),
     ]
 
     operations = [
-        # Migrations related to Project
         migrations.CreateModel(
             name='Project',
             fields=[
-                #Project Model
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('name', models.CharField(help_text='The name of the project.', max_length=200)),
```

Comment:
As requested, this file was created by running makemigrations. It did not need any modification.